apps/frontend/src/network/Routes.tsx

```diff
@@ -19,12 +20,25 @@ const Network: React.FC<{}> = () => {
   const { search } = useLocation();
   const history = useHistory();
   const currentUrlParams = new URLSearchParams(search);
-
+  let filters = currentUrlParams.getAll('filter');
+  const searchQuery = currentUrlParams.get('searchQuery') || '';
+  const [searchQueryDebounce] = useDebounce(searchQuery, 400);
   const onChangeToggle = (pathname: string) => () => {
+    currentUrlParams.delete('filter');
     history.push({ pathname, search: currentUrlParams.toString() });
   };
   const onChangeSearch = (newQuery: string) => {
```

Comment:
These things may also want to move into a hook. But can look at that later when pagination is also in the mix

Reply:
Yeah, I agree but maybe not this PR 😅